Source/WebKit/UIProcess/Launcher/glib/BubblewrapLauncher.cpp

```diff
@@ -752,7 +774,51 @@ static void addExtraPaths(const HashMap<CString, SandboxPermission>& paths, Vect
     }
 }
 
-GRefPtr<GSubprocess> bubblewrapSpawn(GSubprocessLauncher* launcher, const ProcessLauncher::LaunchOptions& launchOptions, char** argv, GError **error)
+#if HAVE(BWRAP_COMMAND_PID)
+static pid_t tryReadCommandPIDFromJSONString(CString string)
+{
+    GRefPtr<JSCContext> context = adoptGRef(jsc_context_new());
```

Comment:
Maybe I should create the JSCContext once per call to readCommandPIDFromJSONStatusFile instead of again and again each time tryReadCommandPIDFromJSONString is called.